src/test/java/by/onliner/tests/PositiveTests.java

```diff
@@ -0,0 +1,44 @@
+package by.onliner.tests;
+
+import by.onliner.driver.DriverManager;
+import by.onliner.dto.Menu;
+import by.onliner.dto.ShoppingCartModalPopup;
+import by.onliner.page.CatalogPageItem;
+import by.onliner.page.MainPage;
+import by.onliner.page.ShoppingCartPage;
+import by.onliner.util.Assertions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PositiveTests extends AbstractTest{
+
+    @Test(description = "Test for removing item from the shopping cart and checking the cart for emptiness")
+    public void removeItemFromShoppingCartTest() {
+        //given
+        MainPage mainPage = new MainPage();
```

Comment:
POs will be used for all the tests for all the test classes, right? Does it mean that we can move their initialization to one place instead of doing it inside each test?

Reply:
Fixed